x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -94,11 +94,13 @@ export class HeadlessChromiumDriverFactory {
     exit$: Rx.Observable<never>;
   }> {
     return Rx.Observable.create(async (observer: InnerSubscriber<any, any>) => {
+      this.logger.debug(`Creating browser driver factory`);
+
       const userDataDir = fs.mkdtempSync(path.join(os.tmpdir(), 'chromium-'));
       const chromiumArgs = args({
         userDataDir,
         viewport,
-        verboseLogging: this.callerLogger.isVerbose,
+        verboseLogging: this.logger.isVerbose,
```

Comment:
`verboseLogging` should just be removed: it is not a valid chromium arg.

I think I want to do that in this PR